boot-qemu.py

```diff
@@ -83,6 +86,12 @@ def parse_arguments():
         help=  # noqa: E251
         "Number of processors for virtual machine. By default, only machines spawned with KVM will use multiple vCPUS."
     )
+    parser.add_argument(
+        "--share-folder",
+        action='store_true',
+        help=  # noqa: E251
+        f"Share {shared_folder} with the guest using virtiofs (requires interactive, not supported with gdb)."
+    )
```

Comment:
Should I be able to use `.` as the share folder?
```
$ boot-utils/boot-qemu.py -i -a x86_64 -k . --share-folder=.
usage: boot-qemu.py [-h] -a ARCH [--append APPEND] [--efi] [-g] [--gdb-bin GDB_BIN] [-i] -k
                    KERNEL_LOCATION [--no-kvm] [-s SMP] [--share-folder] [-t TIMEOUT]
boot-qemu.py: error: argument --share-folder: ignored explicit argument '.'
```

Reply:
```

Reply:
It is a boolean option so omit the `=.`. I can make it customizable but I did not really see much of a reason to.